packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -499,13 +499,15 @@ export default class ImageEdit implements EditorPlugin {
                     this.updateWrapper();
                 }
 
-                updateRotateHandlePosition(
-                    this.editInfo,
-                    this.editor.getVisibleViewport(),
-                    marginVertical,
-                    rotateCenter,
-                    rotateHandle
-                );
+                if (rotateHandle && rotateCenter) {
```

Comment:
@BryanValverdeU I'm doing the check for the rotateHandle and rotateCenter here. If they do exist the `updateRotateHandlePosition` will not be called.

Reply:
Oh I see!
Can we also add to the null check this.editor.getVisibleViewport result? Since the ReturnType is `Rect | null`

Reply:
That is a good observation. I'll add it 👍🏻